conformity/provider.go

```diff
@@ -27,7 +27,7 @@ func Provider() *schema.Provider {
 				Default:  "us-west-2",
 				ValidateFunc: validation.StringInSlice([]string{"eu-west-1", "us-west-2", "ap-southeast-2", "us-1",
 					"in-1", "gb-1", "jp-1", "de-1", "au-1", "ca-1",
-					"sg-1"}, true),
+					"sg-1", "trend-us-1"}, true),
```

Comment:
⛏️ Reminder to not commit this (wonder if we can include it only in dev version somehow?)

Reply:
@johan1252 
Thank you. Will remove this region, only use it for testing.